.github/workflows/sync-versioning-to-cdn.yml

```diff
@@ -0,0 +1,42 @@
+name: CDN config sync
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - build-system/global-configs/versioning.json
+
+jobs:
+  sync:
+    if: github.repository == 'ampproject/amphtml'
+    name: versioning.json
+    runs-on: ubuntu-latest
+    environment: wrangler
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: Install Dependencies
+        run: npm i -g @cloudflare/wrangler
+
+      - name: ⭐ Sync versioning.json to the CDN ⭐
+        run: |
```

Comment:
For maintainability / readability, can we move this business logic into a shell script and invoke it from this file?

Reply:
None of the other files in this directory use this pattern - I agree it's a worthwhile endeavour, maybe we can switch all of them in a followup PR?

Reply:
Absolutely, SGTM. 👍